See also: https://github.com/google/libphonenumber/pull/3903

Not making it optional though, just raising the minimum instead.

---
 cpp/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpp/CMakeLists.txt b/cpp/CMakeLists.txt
index 27e4680ccc..39f05d2c9a 100644
--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -145,7 +145,7 @@ if (USE_BOOST)
   if (WIN32)
     set (Boost_USE_STATIC_LIBS ON)
   endif ()
-  find_package (Boost 1.40.0 COMPONENTS date_time system thread)
+  find_package (Boost 1.69.0 COMPONENTS date_time thread)
   if (NOT Boost_FOUND)
     print_error ("Boost Date_Time/System/Thread" "Boost")
   endif ()
